docs/concepts/geneweaver-data.md

```diff
@@ -1,7 +1,67 @@
 
 GeneWeaver utilizes several external data sources to provide users with a rich set of 
-genomic features and gene associations. These data sources include:
+genomic features and gene associations. 
+
+These data sources include:
 
 - [Ensembl](http://www.ensembl.org/index.html)
-- [NCBI](http://www.ncbi.nlm.nih.gov/)
-- 
\ No newline at end of file
+- [Entrez](https://www.ncbi.nlm.nih.gov/Web/Search/entrezfs.html)
+- [UniGene](https://ncbiinsights.ncbi.nlm.nih.gov/tag/unigene/)
+- [MGI](https://www.informatics.jax.org/)
+- [HGNC](https://www.genenames.org/)
+- [RGD](https://rgd.mcw.edu/)
+- [ZFIN](https://zfin.org/)
+- [FlyBase](https://flybase.org/)
+- [Wormbase](https://wormbase.org/)
+- [SGD](https://www.yeastgenome.org/)
+- [miRBase](http://www.mirbase.org/)
+- [CGNC](https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-10-S2-S5)
+
+
+## Background
+Why bother with all these external data sources? Why not just use one?
+
+In genomics research, there are different typqes of gene and microarray identifiers 
+because the same gene can be referred to by different names or IDs, depending on the 
+context or the source of the information. This can happen even for genomic features 
+within the same species! All these different identifiers can create confusion and make 
+it difficult to compare and integrate data from different sources.
+
+Why are there so many different identifiers, you ask? There are a multitude of reason,
+some of which are:
+
```

Comment:
Can you sum these up in a single sentence per? I don't know if we need all the explanations as they seem to amount to similar outcomes.